app/presenters/person_presenter.rb

```diff
@@ -42,7 +42,13 @@ def speeches
   end
 
   def biography
-    context.govspeak_to_html model.biography
+    if in_current_role?
+      context.govspeak_to_html model.biography
+    else
+      # If a person doesn't currently hold any roles, we use a truncated
+      # version of the biography which ends after the first paragraph
+      context.govspeak_to_html model.biography.split(/\n/).first if model.biography
```

Comment:
I think this needs to be a bit safer, either by splitting the source markdown on newlines, or by actually selecting the first `<p>` tag from the generated HTML. Splitting HTML on whitespace does not guarantee we'll always get the first paragraph as there is no guarantee that the generated HTML doesn't contain whitespace characters elsewhere in the markup.